src/JsonApi/Serializer/ErrorNormalizer.php

```diff
@@ -48,6 +49,12 @@ public function normalize($object, $format = null, array $context = [])
             'description' => $this->getErrorMessage($object, $context, $this->debug),
         ];
 
+        if ($object instanceof FlattenException) {
```

Comment:
This is wrong. We should also support `\Exception`s other than `FlattenException`.

Reply:
I have added a handling for both FlattenException + "normal" Exception, 
I also added a testcase for both.